Makefile

```diff
@@ -1,32 +1,30 @@
 packages = panther_analysis_tool
 
-ci: lint unit integration
+ci:
+	pipenv run $(MAKE) lint unit integration
```

Comment:
does this end up running `pipenv run make lint` -> `pipenv run pipenv run mypy...`? Curious if we should remove that nesting

Reply:
Good catch. That Pipenv run doesn't need to be there. Removed.

Reply:
Interestingly CI fails without this, I'm not sure why yet

Reply:
Resolved. Found a pip install that was not running in the pipenv environment. Corrected and now works as expected